test/runtests.jl

```diff
@@ -43,6 +46,7 @@ Logging.with_logger(quiet ? Logging.NullLogger() : Logging.current_logger()) do
             "force_latest_compatible_version.jl",
             "manifests.jl",
             ]
+            @info "=== Testing `test/$f` ==="
```

Comment:
Should we only print this message if `JULIA_PKG_TEST_QUIET` is set to `false`?

Reply:
That's the case already. It's inside the `with_logger`